.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -118,7 +118,7 @@ jobs:
           fi
           echo ::set-output name=rebuild::yes
       - name: Build and push docker image
-        if: steps.check.outputs.rebuild
+        if: ${{ steps.check.outputs.rebuild }}
```

Comment:
if the condition has operators, even dot `.` is considered as operators, thus the wrapper `${{ }}` is considered nice to have to pass the lint